src/Workspaces/Core/Portable/Utilities/SerializableBytes.cs

```diff
@@ -85,7 +85,77 @@ internal static PooledStream CreateReadableStream(Stream stream, CancellationTok
                     int chunkOffset = 0;
                     while (bytesRemainingForThisChunk > 0)
                     {
-                        int bytesRead = await stream.ReadAsync(chunk, chunkOffset, bytesRemainingForThisChunk, cancellationToken).ConfigureAwait(false);
+                        // In many .NET Framework versions (specifically the 4.5.* series, but probably much earlier
+                        // and also later) there is this particularly interesting bit in FileStream.BeginReadAsync:
```

Comment:
@stephentoub: I'm still assuming it's better to pursue this (since we're aiming this downlevel) than push for the fix on your side of things. That seem reasonable? (If you want to ship a patch I won't complain though...)